api/texts.go

```diff
@@ -0,0 +1,57 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/joschahenningsen/TUM-Live/dao"
+	"github.com/joschahenningsen/TUM-Live/model"
+	"github.com/joschahenningsen/TUM-Live/tools"
+	"net/http"
+	"strconv"
+)
+
+func configTextRouter(router *gin.Engine, wrapper dao.DaoWrapper) {
+	routes := markdownTextRoutes{wrapper}
+	api := router.Group("/api")
+	{
+		api.Use(tools.Admin)
+		api.PUT("/texts/:id", routes.updateText)
+	}
+}
+
+type markdownTextRoutes struct {
+	dao.DaoWrapper
+}
+
+type updateTextDao struct {
+	Name       string `json:"name"`
+	RawContent string `json:"content"`
+	Type       uint   `json:"type"`
```

Comment:
Can you create a type for this? e.g. like here: https://github.com/joschahenningsen/TUM-Live/blob/c8c2d8b540889f9587e71074e2b37eb8082e7551/model/file.go#L9